docs/ref/filterset.txt

```diff
@@ -12,6 +12,8 @@ Meta options
 - :ref:`order_by <order-by>`
 - :ref:`form <form>`
 - :ref:`together <together>`
+- filter_overrides
+- :ref:`strict <strict>`
```

Comment:
I've left `order_by_field` out here, because I got the impression that it's deprecated, but am not sure, especially when looking at https://github.com/carltongibson/django-filter/issues/431#issuecomment-226531373.
/cc @rpkilby